network-api/networkapi/templates/pages/buyersguide/catalog.html

```diff
@@ -26,6 +26,15 @@
       </div>
     </div>
   </div>
```

Comment:
What do you think about moving this into the `guts` block, since we are not showing it in the header anymore?

Reply:
moved